hca/__init__.py

```diff
@@ -1 +1,26 @@
-TWEAK_PROJECT_NAME = "hca"
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import os, logging
+
+import tweak
+
+class HCAConfig(tweak.Config):
+    default_config_file = os.path.join(os.path.dirname(__file__), "default_config.json")
+    @property
+    def config_files(self):
+        return [self.default_config_file] + tweak.Config.config_files.fget(self)
+
+    @property
+    def user_config_dir(self):
+        return os.path.join(self._user_config_home, self._name)
+
+
+_config = None
```

Comment:
why not just:

`_config = HCAConfig(__name__)`

I don't think you get much from lazy-initializing it.

Reply:
I do think it's beneficial to lazy-initialize it, to avoid import-time side effects that include reading a bunch of files and registering an atexit handler that will write a bunch of files.